mono/metadata/icall.c

```diff
@@ -8704,7 +8704,7 @@ ves_icall_System_Threading_Thread_SystemMaxStackSize (void)
 	return mono_thread_info_get_system_max_stack_size ();
 }
 
-ICALL_EXPORT gboolean
+ICALL_EXPORT MonoBoolean
```

Comment:
Is this change intended? Looks like part of #11017.

Reply:
Yeah, it was missed there by accident. It is a separate thing, but it is correct and intended.